test/test_prototype_builtin_datasets.py

```diff
@@ -25,9 +25,10 @@ def extract_datapipes(dp):
     return get_all_graph_pipes(traverse(dp, only_datapipe=True))
 
 
-@pytest.fixture
+@pytest.fixture(autouse=True)
```

Comment:
Although this patch is now applied by default, if required, a test can still request this fixture the normal way.